src/client/apis/csiortc/csiostats/CsioStats.js

```diff
@@ -114,6 +120,31 @@ class CsioStats {
     };
     TriggerEvent(CsioEvents.CsioStats.ON_INITIALIZED, detail);
   }
+  csPreCallTestResultsCallback(status, results) {
+    if (status === this.csObject.callStatsAPIReturnStatus.success) {
+      const precallStats = {
+        connectivity: results.mediaConnectivity,
+        rtt: results.rtt,
+        loss: results.fractionalLoss,
+        throughput: results.throughput
+      };
```

Comment:
whats the point of this mapping?

Reply:
No big point for this application. 😄 Usually, I always use mapping to decouple the API object struct from UI/Application object struct. In case the API object structure change, we only need to change the mapping, so that UI/application logic don't have to worry about the changes. 
I agree, it is redundant for this application.